channels/ae.m3u

```diff
@@ -37,8 +37,8 @@ https://dmitwlvvll.cdn.mangomolo.com/dubaisportshd/smil:dubaisportshd.smil/index
 https://dmitwlvvll.cdn.mangomolo.com/dubaisportshd5/smil:dubaisportshd5.smil/index.m3u8
 #EXTINF:-1 tvg-id="Dubai TV ARB" tvg-name="Dubai TV ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/dWueFIE.png" group-title="",Dubai TV
 https://dmisxthvll.cdn.mangomolo.com/dubaitv/smil:dubaitv.stream.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/Q1Kjknl.png" group-title="Music",Eamar Music
-http://82.212.74.3:8000/live/7514.m3u8
+#EXTINF:-1 tvg-id="Etisalat Dubai Zaman TV ARB" tvg-name="Etisalat Dubai Zaman TV ARB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/F5U9qpf.png" group-title="",Dubai Zaman
+https://dmiffthvll.cdn.mangomolo.com/dubaizaman/smil:dubaizaman.smil/playlist.m3u8
```

Comment:
The link doesn't show a picture, just a sound. Tested in Elmedia Video Player and VLC player.

Reply:
Weird, it looks like the low quality stream is broken but the highest quality one works.

https://streamtest.in/logs/https-dmiffthvll-cdn-mangomolo-com-dubaizaman-smil-dubaizaman-smil-chunklist_b1500000-m3u8

I think the host is having problems and maybe it will fix them eventually.